.github/workflows/update-foundry-prove.yml

```
@@ -0,0 +1,27 @@
+name: 'Update Foundry prove'
+on: push
+jobs:
+  update-output:
+    name: 'Update Foundry expected output'
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: [self-hosted, linux, huge]
```

This is to run it on our big runners, not the default Ubuntu runners github supplies.

---

If it was desired to run on github @ehildenb that's on me. I instructed @iFrostizz to run on private runners to support the more compute intensive executions he was working on. I'm not familiar if this is that specific workflow so please disregard if public is preferred here.